styles/contactUs.css

```diff
@@ -0,0 +1,216 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
Sprawdz, proszę czy to na pewno nie psuje nam żadnej innej sekcji

Reply:
Sprawdziłem...lekko wpływa :)
Zmieniłem selektor na section.contactUs *